governance/votes/Executive vote - November 28, 2024.md

```diff
@@ -0,0 +1,267 @@
+- **Authorization**: [Atlas A.5.6](https://sky-atlas.powerhouse.io/A.5.6_Launch_Project/1f433d9d-7cdb-406f-b7e8-f9bc4855eb77%7C8d5a)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/utilization-of-the-launch-project-under-the-accessibility-scope/21468/26)
+
+If this executive proposal passes, then the following Launch Project funding distribution will be made:
+
+- Transfer **10 million USDS** to the Launch Project at [0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F](https://etherscan.io/address/0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F).
+- Transfer **24 million SKY** to the Launch Project at [0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F](https://etherscan.io/address/0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F).
+
+### Sky Ecosystem Liquidity Bootstrapping
+
+- **Authorization**: [Poll 1158](https://vote.makerdao.com/polling/QmYHUDVA)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/utilization-of-the-sky-ecosystem-liquidity-bootstrapping-budget-a-5-6-1-9/25537/2)
+
+If this executive proposal passes, then the following Sky Ecosystem Liquidity Bootstrapping funding distribution will be made:
+
+- Transfer **6 million USDS** to [0xD8507ef0A59f37d15B5D7b630FA6EEa40CE4AFdD](https://etherscan.io/address/0xD8507ef0A59f37d15B5D7b630FA6EEa40CE4AFdD).
+
+### Integration Boost Funding
+
+- **Authorization**: [Poll 1158](https://vote.makerdao.com/polling/QmYHUDVA)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/utilization-of-the-integration-boost-budget-a-5-2-1-2/25536/2)
+
+If this executive proposal passes, then the following Integration Boost funding distribution will be made:
+
+- Transfer **3 million USDS** to [0xD6891d1DFFDA6B0B1aF3524018a1eE2E608785F7](https://etherscan.io/address/0xD6891d1DFFDA6B0B1aF3524018a1eE2E608785F7).
+
+### ESM Minimum Threshold Increase
+
+- **Authorization**: [Governance Facilitator Authorization](https://forum.sky.money/t/november-28th-spell-parameter-proposal-esm-threshold-gsm-delay-increase/25579/2)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/november-28th-spell-parameter-proposal-esm-threshold-gsm-delay-increase/25579)
+
+If this executive proposal passes, then the [Emergency Shutdown Module (ESM)](https://docs.makerdao.com/smart-contract-modules/shutdown/emergency-shutdown-module) [Minimum Threshold (`min`)](https://manual.makerdao.com/module-index/module-emergency-shutdown#minimum-threshold-min) will be increased by 200,000 MKR from 300,000 MKR to **500,000 MKR**.
+
+
+### Aligned Delegate Compensation for October 2024
+
+- **Authorization**: [Atlas A.1.5.8](https://sky-atlas.powerhouse.io/A.1.5.8_Budget_For_Prime_Delegate_Slots/e3e420fc-9b1f-4fdc-9983-fcebc45dd3aa%7C0db3af4ece0c)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/october-2024-aligned-delegate-compensation/25581)
+
+If this executive proposal passes, then a total of **20,701 USDS** will be distributed to six Aligned Delegates and their teams as compensation for October 2024.
+
+| Delegate | Amount (USDS) | Address |
```

Comment:
For verification, the ordering here should match the order on the Exec sheet. Non-blocking